src/main/java/net/authorize/sample/SampleCode.java

```diff
@@ -138,15 +138,16 @@ private static void RunMethod(String methodName)
     {
         // These are default transaction keys.
         // You can create your own keys in seconds by signing up for a sandbox account here: https://developer.authorize.net/sandbox/
-        String apiLoginId           = "5KP3u95bQpv";
-        String transactionKey       = "346HZ32z3fP4hTG2";
+        String apiLoginId           = "mbld_api_%63ty4Rq";
+        
+        String transactionKey       = "123abc";
```

Comment:
Please revert this change. The original credentials work in the Sandbox environment and the new credentials work only in a test environment.

Reply:
Revert the changes as per the original credentials work in the Sandbox environment.